Command/README.md

```diff
@@ -0,0 +1,71 @@
+#  thin-edge.io Command plugin
+
+To use command on a device that runs thin-edge.io, a plugin of the operation plugin concept is used. The tedge_agent is checking for command operation and is triggering the particular plugin. You can use the Shell tab in device management to use a command. Please be aware of the power of such an feature. Somebody with access could start various shell commands. Its more designes as an idea on how the plugin mechanism works.
+
+## Requirements
+
+- Working thin-edge.io installation
+
+- Python3 and pip3 installation (will not work on python2)
+
+
+## Installation 
+
+1. Clone this repo on the thin-edge.io device
+2. run pip3 install -r requirements.txt from this Command directory
+3. Copy c8y_Command to the following directory "/etc/tedge/operations/c8y/"
+4. Copy c8y_Command.py to the following directory "/bin/"
+5. Make sure, that both files do have permissions for beeing executed by tedge_mapper ("chmod 777 FILENAME")
+
+
+## Usage
+
+Make sure thin-edge.io is connected to Cumulocity.
+If installation is done properly accoording to the steps above, you hae to disconnect and reconnect thin-edge.io. In that way the suppoertedOperations will be updated.
+
+```shell
+sudo tedge disconnect c8y
+```
+
+and
+
+```shell
+sudo tedge connect c8y
+```
```

Comment:
This works, but only the `tedge_mapper` daemon needs to be restarted.

Aside question:
* Would an [auto reload feature](https://github.com/thin-edge/thin-edge.io/issues/612) be helpful?

Reply:
It definitely would!